en/install.md

```diff
@@ -52,64 +52,38 @@ The proxy will listen on all interfaces on port 443 (with a redirect from 80).
 
 ## Database
 
-To initially configure the Snap Store Proxy, you will need either a
-database already prepared, or the ability to install PostgreSQL locally. 
+Ensure that proper PostgreSQL database, user and database extensions are set up.
+This can be done by adjusting the following script to your needs and running it
+using `psql` as your PostgreSQL server **superuser**:
 
-### Prepared database
+    CREATE ROLE "snapproxy-user" LOGIN CREATEROLE PASSWORD 'snapproxy-password';
 
-Choose this approach if you are experienced with PostgreSQL and want to setup a
-production ready installation.
+    CREATE DATABASE "snapproxy-db" OWNER "snapproxy-user";
 
-You will need to create a database:
+    \connect "snapproxy-db"
 
-    CREATE DATABASE dbname;
+    CREATE EXTENSION "btree_gist";
 
-And a new user:
+### Example database setup
 
-    CREATE ROLE user LOGIN CREATEROLE PASSWORD 'password';
+Simple local Ubuntu setup can look like this:
 
-The user needs to be able to create objects in the database:
+1. Install postgresql
 
-    GRANT CREATE ON DATABASE dbname TO user;
+        sudo apt install postgresql
 
-A prepared database must have the 
-[btree_gist](https://www.postgresql.org/docs/current/static/btree-gist.html) 
-extension installed. This extension requires superuser privileges to create.
-You should connect to DBNAME as a superuser and create the extension:
+2. Save the above PostgreSQL script as proxydb.sql and run it:
 
-    CREATE EXTENSION btree_gist;
+        sudo -u postgres psql < proxydb.sql
```

Comment:
Can we ship this script in the snap? And use it to set up our functional test db, so we know it works? Not a blocker for this of course, but I think that's better than telling the user to 'save the above' manually. I mean, we have to automate for functional tests anyway, so why not make it a first class user tool/docs?

Reply:
Thanks! That's a good idea. I'll try to address this separately.